src/main/java/SellStockUseCase/SellStockController.java

```diff
@@ -0,0 +1,17 @@
+package SellStockUseCase;
+
+import java.io.IOException;
+public class SellStockController {
+    private final SellUseCaseInteractor interactor;
+
+    public SellStockController() {
+        interactor = new SellUseCaseInteractor();
+    }
+
+    public SellOutputResponse sellStock(SellInputRequest sell) throws IOException {
+        SellUseCaseInteractor sellUseCaseInteractor = new SellUseCaseInteractor();
```

Comment:
what's the point of making another interactor, u already instantiate one in the constructer

Reply:
Don't know why I did that while coding. I just changed it.